src/EnergyPlus/HeatRecovery.cc

```diff
@@ -4488,35 +3656,20 @@ namespace HeatRecovery {
         // The range of each independent variable is provided by the user and are based on the limits of the
         // empirical model. These limits are tested in this subroutine each time step and returned for use by the calling
         // routine.
-        // REFERENCES:
-        // na
 
         // Using/Aliasing
         auto &SysTimeElapsed = state.dataHVACGlobal->SysTimeElapsed;
         auto &TimeStepSys = state.dataHVACGlobal->TimeStepSys;
         using General::CreateSysTimeIntervalString;
 
-        // Locals
-        // SUBROUTINE ARGUMENT DEFINITIONS:
-        // regen outlet humidity ratio equation
-
-        // SUBROUTINE PARAMETER DEFINITIONS:
-        //  CHARACTER(len=*), PARAMETER :: OutputFormat  ='(F10.6)'
-
-        // INTERFACE BLOCK SPECIFICATIONS:
-        // na
-
-        // DERIVED TYPE DEFINITIONS:
-        // na
-
         // SUBROUTINE LOCAL VARIABLE DECLARATIONS:
-        auto &OutputChar = state.dataHeatRecovery->OutputChar2;
-        auto &OutputCharLo = state.dataHeatRecovery->OutputCharLo2;
-        auto &OutputCharHi = state.dataHeatRecovery->OutputCharHi2;
-        auto &CharValue = state.dataHeatRecovery->CharValue2;
-        auto &TimeStepSysLast = state.dataHeatRecovery->TimeStepSysLast2;
-        auto &CurrentEndTime = state.dataHeatRecovery->CurrentEndTime2;
-        auto &CurrentEndTimeLast = state.dataHeatRecovery->CurrentEndTimeLast2;
+        auto &OutputChar = state.dataHeatRecovery->error2.OutputChar;
```

Comment:
Afraid to ask what these are, but 99.23% sure they should be values and not references.

Reply:
OutputChar is a string so I'd avoid a value copy there.  But either way, I don't think it's worth modifying these right now.  I'd really like to do a pass over all this recurring error stuff.  After the SolveRoot pass.  After wrapping enums and SCV.

Reply:
See above comment. Feel free to respond to anything that would hold up the PR with 📌 .